cimsparql/sparql/three_winding_dummy_nodes.sparql

```diff
@@ -1,7 +1,7 @@
 # Name: Three winding with dummy nodes
 PREFIX cim:<${cim}>
 PREFIX SN:<${SN}>
-select ?node ?name (?name as ?busname) ?un ?station ?bidzone ?island (False as ?is_swing_bus)
+select ?node ?busname ?substation ?un ?station ?bidzone ?island (False as ?is_swing_bus)
```

Comment:
**Just a tought**: here we implicitly create a node/substation? For readability of the query should we use good variable names internally that reflect what the variable refers to and do the transition in the select statement?

For examle:
Instead of

```sparql
?container cim:IdentifiedObject.mRID ?substation;
```

we use

```sparql
?container cim:IdentifiedObject.mRID ?container_mrid;
```

and in the select statement

```
# Create an artificial substation by using properties of the Equipment container
select ... (?container_mrid as ?substation_mrid)
```